easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Satsuma2'
+local_commit = '37c5f38'
+version = '2022-03-04'
+
+homepage = 'https://github.com/bioinfologics/satsuma2'
+description = """Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences
+providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to
+accomodate the billions of base pairs in vertebrate genomes)."""
```

Comment:
```suggestion
accommodate the billions of base pairs in vertebrate genomes)."""
```

Reply:
Nice catch. This was copy-pasted from https://github.com/bioinfologics/satsuma2/blob/master/README.md, so maybe send them a PR too? ;)